ansible/playbooks/deploy-prometheus-grafana.yml

```diff
@@ -29,7 +29,7 @@
   roles:
   - cloudalchemy.grafana
   vars:
-    grafana_version: 7.5.10
+    grafana_version: latest
```

Comment:
Just to check, have you verified that our grafana dashboard works with the latest version?

Reply:
I pulled up grafana and imported the dashboard and saw that everything looked fine. BUT i had to go through the process of setting up grafana and prometheus manually ahead of time and had nothing going on the cluster. The charts were empty which was what i expect, and figured importing the charts json that was generated would be what caused a problem, so without doing additional testing of creating topics, producing/consuming, and it not giving an error after hooking up prometheus I deemed the addl steps unnecessary. LMK if you want me to do addl testing.

Reply:
ok, i'll look at what their latest version number is and try it out. I'm also in the process of having prometheus/grafana installed along with it, and will put some load on it to make sure the charts draw.

Reply:
TIL that i should have expected grafana and prometheus to get installed and dashboard automagically imported OOB, but didn't b/c I was trying to use the same host for both monitoring and redpanda. I am re-running tests in this fork, stand by..

Reply:
changed to and tested with pinning to Grafana *9.0.0* (latest) and it deployed fine, imported fine, all charts rendered actively.